patroni/dcs.py

```diff
@@ -216,8 +244,29 @@ def attempt_to_acquire_leader(self):
     def set_failover_value(self, value, index=None):
         """Create or update `/failover` key"""
 
-    def manual_failover(self, leader, member, index=None):
-        return self.set_failover_value(leader + (':' + member if member else ''), index)
+    def manual_failover(self, leader, member, planned_at=None, index=None):
```

Comment:
That may be true, however calling `localize_datetime` on an already timezone returns the same value.
It was added to ensure that whenever we write a value, we attach a timezone to it, even though currently we don't have anyone calling the function with unaware timezones.